lua/wrlt/plugin/init.lua

```diff
@@ -105,21 +105,12 @@ lazy.setup({
                 'williamboman/mason-lspconfig.nvim',
             },
             config = function()
+                require('wrlt.jdtls').setup()
```

Comment:
Setting up wrlt.jdtls will require vim-dap to allow debugging. There is no dependency with that plugin.

Reply:
180eb23 includes the dependency.